core/src/main/res/values/strings.xml

```diff
@@ -346,8 +345,6 @@
     <string name="pref_gpodnet_logout_toast">Logout was successful</string>
     <string name="pref_gpodnet_setlogin_information_title">Change login information</string>
     <string name="pref_gpodnet_setlogin_information_sum">Change the login information for your gpodder.net account.</string>
-    <string name="pref_playback_speed_title">Playback Speeds</string>
-    <string name="pref_playback_speed_sum">Customize the speeds available for variable speed audio playback</string>
```

Comment:
Dang. You'd think that AS/Gradle (after taking forever to bundle a few files together) would complain if a string is missing in the default language m(
Not even Lint does catch that, or does it? ("Incomplete translation" does not check the default translation)